source/includes/harvest/_demographic_data.md

```diff
@@ -0,0 +1,463 @@
+# Demographic Data
+
+Greenhouse Inclusion ...
+
+## The Demographic Question object
+
+```json
+{
+  "id": 123,
+  "translations": [
+    {
+      "language": "en",
+      "name": "What is your favorite color?"
+    }
+  ]
+}
+```
+
+### Noteworthy attributes
+
+| Attribute | Description |
+|-----------|-------------|
+| id | The demographic question's unique identifier |
+| translations.language | Only `en` (English) is supported at this time.
+| translations.name | The question text
+| active | If `false`, the question has been deleted.
+
+## The Demographic Answer Option object
+
+```json
+{
+  "id": 456,
+  "free_form": false,
+  "active": true,
+  "demographic_question_id": 123,
+  "translations": [
+    {
+      "language": "en",
+      "name": "Blue"
+    }
+  ]
+}
+```
+
+### Noteworthy attributes
+
+| Attribute | Description |
+|-----------|-------------|
+| id | The demographic answer option's unique identifier |
+| translations.language | Only `en` (English) is supported at this time.
+| translations.name | The answer option text.
+| active | If `false`, the answer option has been deleted.
+| free_form | If `true`, the answer option allows free-form user input.
+| demographic_question_id | The demographic question for which the answer option belongs to.
+
+## The Demographic Answer object
+
+```json
+{
+  "id": 51,
+  "free_form_text": "12am",
+  "application_id": 107594341,
+  "demographic_question_id": 25,
+  "demographic_answer_option_id": 106,
```

Comment:
If `"free_form_text"` is not null, will `"demographic_answer_option_id"` ever have a value?

Reply:
Yes.  Every answer will have a `demographic_answer_option_id`.  A free-form answer will point to a demographic answer option of the `free-form` type.